modules/xfem/include/userobjects/MeshCut3DUserObject.h

```diff
@@ -30,8 +31,11 @@ class MeshCut3DUserObject : public GeometricCutUserObject
 
```

Comment:
Since this cut UO is specific to represent cracks, and maybe we should change the name to `CrackMeshCut3DUserObject`, or something else. Then we can develop other MeshCut3DUserObject to represent material interface.